### DISCUSSION TITLE
why can't find the some configuration items in the official documentation?

### DISCUSSION THREAD

**Discussion by heshiyingx** [apache/rocketmq#6838] [Q&A]
Why I can't find the description of the storage location configuration items in the official documentation, such as: storePathRootDir, storePathCommitLog, storePathIndex, storeCheckpoint, abortFile, etc.

documentation url:https://rocketmq.apache.org/docs/4.x/parameterConfiguration/02server.

thanks everyone

**Comment by chenzlalvin** [comment]
The official website documentation update is not yet complete, some parameters have not been supplemented, and some parameters will only take effect in the 5.x version.

**Reply by heshiyingx** [reply]
Thank you for your answer. But those parameters above are from version 4.x. I would like to know if those parameters are deprecated? Can they continue to be used?
